app/javascript/components/widgets/widgets/climate/soil-organic/selectors.js

```diff
@@ -119,9 +119,9 @@ export const parseSentence = createSelector(
         }
       };
     }
-    const iso = Object.keys(locationsDict).find(
-      key => locationsDict[key] === location
-    );
+    const iso =
+      locationsDict &&
```

Comment:
good catch! I missed a couple of these

Reply:
They were breaking hot reload so needed fixing.